sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -155,13 +155,15 @@ export interface DownloadBlobResult {
 
 // @public
 export interface DownloadManifestOptions extends OperationOptions {
+    mediaType?: string;
```

Comment:
Is there an easy way to pass in multiple values?  .NET and Java have added overloads that take a collection of accept header media types so callers don't have to concatenate strings themselves.

Reply:
Oh yeah, we can put in an overload for that easy.